devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -214,11 +212,6 @@ public void execute() throws MojoExecutionException {
             success = createProject.execute().isSuccess();
 
             File createdDependenciesBuildFile = new File(projectRoot, buildToolEnum.getDependenciesFile());
-            if (success) {
-                success = new AddExtensions(QuarkusProject.of(projectFolderPath, platform, buildToolEnum))
-                        .extensions(extensions).execute()
-                        .isSuccess();
-            }
```

Comment:
Why remove the ability to add extensions when you create a project?
I find the feature quite useful.

Reply:
@stalep it's done directly from the create command (no need to call two separate commands)